src/App.js

```diff
@@ -1,25 +1,86 @@
-import logo from './logo.svg';
-import './App.css';
+//import { Routes, Route } from "react-router-dom";
```

Comment:
Usually before merging I would recommend to delete all commented out code so that the main branch is as clean as possible.